src/js/actions/layers.js

```diff
@@ -740,13 +740,19 @@ define(function (require, exports) {
                 if (firstUninitializedLayers.size === 50) {
                     this.whenIdle(initializeLayersBackground, document);
                 }
+            })
+            .error(function (err) {
+                /* jshint multistr: true */
+                log.error("InitializeLayersBackground action failed for document %s, \
```

Comment:
http://bluebirdjs.com/docs/api/error.html

I specifically wanted to log on operationalErrors fail. If you still think that's a bad idea, I will move to `catch`.